lib/Backend/JITTimePolymorphicInlineCacheInfo.cpp

```diff
@@ -53,8 +53,9 @@ JITTimePolymorphicInlineCacheInfo::InitializePolymorphicInlineCacheInfo(
     __in Js::PolymorphicInlineCacheInfo * runtimeInfo,
     __out PolymorphicInlineCacheInfoIDL * jitInfo)
 {
+#pragma warning(suppress: 6001)
```

Comment:
sure, I can do it in next commit

---
In reply to: [92459896](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92459896) [](ancestors = 92459896,92458452,92454774)